src/components/Card/CardSecondary/CardSecondary.tsx

```diff
@@ -0,0 +1,143 @@
+import styled from "styled-components";
+import { Badge } from "@/components/Badge/Badge";
+import { Title } from "../../Typography/Title/Title";
+import { Text } from "../../Typography/Text/Text";
+import { IconName } from "@/components/Icon/types";
+import { Icon } from "../..";
+
+export type BadgeState =
+  | "default"
+  | "success"
+  | "neutral"
+  | "danger"
+  | "disabled"
+  | "warning"
+  | "info";
+
+export interface CardProps {
+  title: string;
+  image: IconName;
+  hasBadge?: boolean;
```

Comment:
i would remove this property and use only `badgeText`. If `badgeText` has any value then we show it

Reply:
Done.